.clang-tidy

```diff
@@ -0,0 +1,37 @@
+Checks: >
+    '-*,
```

Comment:
Shouldn't have a single quote here or at the end

Reply:
updated